### ISSUE TITLE
Bug reported in google group: cannot coerce type closure to vector of type character

### ISSUE THREAD

**Issue by jhmigueles** [wadpac/GGIR#846]
Some users have reported the same error in the google group. It seems to appear at the moment of the generation of the activity log, and it appears in different GGIR versions (including 2.9-4). I cannot replicate the error for now, so I will create a branch to investigate this.

**Comment by jhmigueles** [comment]
Hi again @tarensanders. I have loaded the dplyr package and didn't get any error. Can you try installing the branch "issue846_bugCharacterGoogleGroup" and see if the error is still there?

You can do that with `remotes::install_github("wadpac/GGIR", ref = "issue846_bugCharacterGoogleGroup")`

**Comment by tarensanders** [comment]
Hi @jhmigueles - you're right it passes fine. Just to make sure I wasn't going crazy, I loaded an old version of GGIR (2.5-1, which was a version we used for a project) and that version does fail the chainof5parts test if dplyr is loaded. Looks like it's been fixed in the many versions since.

Sorry for the bum steer. When I saw this error it occurred to me that I never filed a bug for the dplyr issue.
